src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsInterstitialObject.java

```diff
@@ -30,17 +33,32 @@
   @FindBy(css = "iframe.wikia-ad-iframe")
   private WebElement interstitialAdIframe;
 
-  @FindBy(css = "div#ext-wikia-adEngine-template-modal, div.lightbox-content-inner > div")
+  static final String
+      INTERSTITIAL_AD_WRAPPER_SELECTOR = "div#ext-wikia-adEngine-template-modal, div.lightbox-content-inner > div";
+
+  @FindBy(css = INTERSTITIAL_AD_WRAPPER_SELECTOR)
   private WebElement interstitialAdWrapper;
 
   @FindBy(css = ".lightbox-close-wrapper, #ext-wikia-adEngine-template-modal .close")
   private WebElement interstitialCloseButton;
 
+  @FindBy(css = ".lightbox-close-wrapper")
+  private WebElement interstitialCloseButtonMercury;
+
+  @FindBy(css = "#ext-wikia-adEngine-template-modal .close-counter")
+  private WebElement interstitialCounterOasis;
+
   public AdsInterstitialObject(WebDriver driver, String testedPage,
                                Dimension resolution) {
     super(driver, testedPage, resolution);
   }
 
+  public void waitForInterstitialShowUp() {
+    JavascriptActions javascriptActions = new JavascriptActions(driver);
+    javascriptActions.waitForJavaScriptTruthy("document.querySelectorAll(\"" + INTERSTITIAL_AD_WRAPPER_SELECTOR
```

Comment:
Minor thing: how about using `String.format()` and use `'` within `querySelectorAll()` to make it more readable with fewer slashes? ;)